kotlin-native/build-tools/build.gradle.kts

```diff
@@ -42,11 +42,8 @@ repositories {
     maven(kotlinCompilerRepo)
     maven(buildKotlinCompilerRepo)
     maven("https://cache-redirector.jetbrains.com/maven-central")
```

Comment:
Cleanup cache-redirector, also we add jcenter to all projects in the root buildscript

Reply:
for some reasons repositoryCheck fails on this.